### ISSUE TITLE
manager: progress indicator when installing Python modules

### ISSUE THREAD

**Issue by OneCDOnly** [OneCDOnly/sherpa#69]
I've no-idea how I'm going to do this, but it would be nice to have. :grin:

**Comment by OneCDOnly** [comment]
This is a bit less important now, as the `pip` modules are installed only once - after Entware is installed. They're not added again unless Entware is reinstalled.

**Comment by OneCDOnly** [comment]
Closing for now as I can't think of a way to do this.
